src/index.js

```diff
@@ -0,0 +1,9 @@
+import {
+  restaurantPage,
+} from "./helpers";
+
+import {
+  homePageContent,
+} from "./home";
+
+window.addEventListener("load", () => restaurantPage.generateInitialContent(document.getElementById("content"), ["Home", "About", "Contact"], homePageContent));
```

Comment:
Do you need the event listener for load?

Reply:
Hi Alvaro,

Thanks. Per our message in this Pull Request we didn't need to export a function from each tab because we already have configured the content in such a way that we only need a reusable function to generate the HTML for each tab (it's in the helper.js file).. As for the event listener for the window.. Sure we can remove it, we'll simply call the generateInitial content in the index.js file :) 

By the way, how does this review process go? Do we need to create a new review request for the reviewer to read our answers to his/her questions?